src/Http/Controllers/RecentJobsController.php

```diff
@@ -38,6 +38,8 @@ public function index(Request $request)
         $jobs = $this->jobs->getRecent($request->query('starting_at', -1))->map(function ($job) {
             $job->payload = json_decode($job->payload);
 
+            $job->delayed = strtotime(optional(unserialize($job->payload->data->command))->delay);
```

Comment:
If you are unserializing the entire command to get this won't that re-issue the database queries needed to get the Eloquent models / collections required by the job?

Reply:
@taylorotwell Just tried, indeed it will introduce queries to be run whenever a model is type hinted into the job class, this will introduce reduced performance once there are jobs with a lot of models.

Is there a way to disable this magic once we call an unserialise on the payload?

Reply:
Not that I can think of off of the top of my head. It may be possible to extract the delay from the raw serialized string but I'm not sure.

Reply:
I am going to think and try some stuff out. Will get back on this, thanks, good find 👍